packages/financial-templates-lib/test/price-feed/DominationFinancePriceFeed.js

```diff
@@ -208,9 +208,9 @@ contract("DominationFinancePriceFeed.js", function() {
     assert.isTrue(await invertedPriceFeed.update().catch(() => true), "Update didn't throw");
 
     assert.equal(priceFeed.getCurrentPrice(), undefined);
-    assert.throws(() => priceFeed.getHistoricalPrice(earliestTick));
+    await priceFeed.getHistoricalPrice(earliestTick).then(() => assert.fail());
```

Comment:
@nicholaspai this is the syntax that we have to use to replicate `.throws` since `.throws` doesn't handle async functions (at least I don't _think_ it does). It's a little harder to understand, but at least it's not super painful to write.